README.md

```diff
@@ -116,7 +116,7 @@ Import the following dependency in your Vertx project:
 
 * **org.pac4j.vertx.impl.StatefulPac4jAuthProviderImpl** implementation of the vert.x AuthProvider interface for stateful (session-based) Pac4j handling
```

Comment:
These auth providers have been removed, haven't they? Replaced by the `Pac4jAuthProvider`?